Src/BootCamp.Chapter/Computer/Body.cs

```diff
@@ -6,5 +6,15 @@ namespace BootCamp.Chapter.Computer
 {
     public class Body
     {
+        private string _manufacturersId;
+        public Body(string manufacturersId)
+        {
+            _manufacturersId = manufacturersId;
+        }
+
+        public string GetManufacturersId()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetManufacturersId' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5121609)